### PR TITLE
Enforce license header for all files

### PR DESCRIPTION
**Pull request by mik-laj**
Hello,

Some files did not have the required license header so I added an auto check.

Best regards,
Kamil Breguła

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+default_stages: [commit, push]
+minimum_pre_commit_version: "1.20.0"
+repos:
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
```
Comment (soupytwist) [inline .pre-commit-config.yaml:19]: This doesn't resolve -- what should it be?

**Comment by soupytwist** [comment]
Cool, thanks for adding this!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (mik-laj) [reply .pre-commit-config.yaml:19]: It works for me. https://github.com/Lucas-C/pre-commit-hooks
Similar code exists in other Google projects: 
https://github.com/google/triage-party/blob/master/.pre-commit-config.yaml
https://github.com/GoogleCloudPlatform/oozie-to-airflow/blob/master/.pre-commit-config.yaml

Comment (soupytwist) [reply .pre-commit-config.yaml:19]: Weird, I could swear I got a 404 yesterday. Sorry then!

**Pull request merged**